### ISSUE TITLE
Training convergence on colab

### ISSUE THREAD

**Issue by kgb0255** [changhoonhahn/gqp_mc#55]
![image](https://user-images.githubusercontent.com/57269223/98594981-3d0c9c80-228a-11eb-8b95-64d2dea102a2.png)
y axis: loss, x axis: epoch.

Recent training on colab might suggest that we need a longer training, although the y-axis unit has to be checked again. The validation loss is not retained here, so I will run additional training to check the unit. Above plot is generated from complexdust, 50 pca components, wavebin 0, [300,300,300] hidden layers.

**Comment by changhoonhahn** [comment]
This issue was because list of loss values were reset at every learning rate. Since this has been resolved, I'm closing this issue.
